### PR TITLE
feat(helm): helm chart v2

### PR DESCRIPTION
**Pull request by kuikuikuizzZ**
**What type of PR is this?**
/kind feature
/area training
-->

**What this PR does / why we need it**:

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @whalecold @tskdsb 

**Does this PR introduce a user-facing change?**:

```release-note
Add v2 helm chart
```

### CODE REVIEW

File: manifests/tf-operator/templates/daemonsets.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{{- range $dsName, $ds := .Values.daemonSets }}
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ $dsName }}
+  namespace: {{ $.Release.Namespace | quote }}
```
Comment (whalecold) [inline manifests/tf-operator/templates/daemonsets.yaml]: 这个文件不需要了吧

**Comment by whalecold** [comment]
你看下有些不需要的资源全部删了吧

File: manifests/tf-operator/templates/daemonsets.yaml (hunk 1 above)
Comment (kuikuikuizzZ) [reply manifests/tf-operator/templates/daemonsets.yaml]: Done

**Pull request merged**